protocol/src/main/resources/protocol.xml

```diff
@@ -60,6 +60,7 @@
       <validValue name="ALREADY_EXISTS">2</validValue>
       <validValue name="INVALID_STATE">3</validValue>
       <validValue name="PROCESSING_ERROR">4</validValue>
+      <validValue name="EXCEEDED_BATCH">5</validValue>
```

Comment:
:wrench: We could change the name of the rejection type to `EXCEEDED_BATCH_RECORD_SIZE`.

Currently, we print only the rejection type in the client exception. If the rejection type is more self-descriptive then the exception is easier to understand.

:thinking: In the configuration, the size is called max. message size. But I think the term `message` could be confusing. The underlying protocol is based on messages but we try to write a batch of records.